CHANGELOG.md

```diff
@@ -1,4 +1,82 @@
-## [Unreleased][unreleased]
+# Table of Contents
+
+- [Planned](#planned)
+    - [0.12.0](#0120---tbd)
+- [Scheduled](#scheduled)
+    - [0.11.2](#0112---20171129)
+- [Released](#released)
+    - [0.11.1](#0111---20171024)
+    - [0.10.4](#0104---20171024)
+    - [0.11.0](#0110---20170816)
+    - [0.10.3](#0103---20170524)
+    - [0.10.2](#0102---20170501)
+    - [0.10.1](#0101---20170327)
+    - [0.10.0](#0100---20170307)
+    - [0.9.9 and prior](#099---20170202)
+
+# Planned
+
+This section describes planned releases of Kong and their thematic fomr a
+high-level perspective. Those releases do not have a fixed release date yet.
+
+## [0.12.0] - TBD
+
+This release will mainly focus around two new major features:
+
+- The introduction of **health checks** in the load-balancer
```

Comment:
Can we link this to Issue(s)? Given the inherent uncertainty about things happening the in future, it is more like a link that says, and may we should be explicit, "Hey, we think this will at least partly address issues 1, 2, and 3 - but hey, no promises :)"